### ISSUE TITLE
Text placed above counts like a new word

### ISSUE THREAD

**Issue by danielrruf** [gcelano/MALP#1]
Apparently, text placed over the baseline (<add place="above">) is identified as a new word in MALP. See Basp.47.46 r.6

**Comment by gcelano** [comment]
Yes, this is known and I document this in the forthcoming paper about the data. This is a limitation of the data
